kratos/processes/levelset_convection_process.h

```diff
@@ -831,13 +848,11 @@ class LevelSetConvectionProcess
         if (ElementType == "levelset_convection_supg") {
             default_parameters = Parameters(R"({
                 "dynamic_tau" : 0.0,
-                "cross_wind_stabilization_factor" : 0.7,
-                "requires_distance_gradient" : false
```

Comment:
I would not remove this field. Indeed, it was intentionally added to enforce adding it for any new element. This would imply checking another field if a new element, either in ours or in a fork side, is added.
The point was to check the gradient calculation with this (not with formulation related info like the `include_anti_diffusivity_terms`.

Reply:
> > Which are the necessary changes? Introducing the gradient calculation inside the levelset convection?
> 
> Yes. Updating the nodal gradient inside the (substepping) loop.

So far we did it once per time step right?

Reply:
So, you mean that `include_anti_diffusivity_terms` and  `requires_distance_gracient` are redundant right?

Reply:
For this specific element, anti_diffusivity_terms require distance_gradient. Also, the limiter (elemental) will be needed to bound the anti-diffusive term.